tests/test_application.py

```diff
@@ -267,3 +267,42 @@ def helper(scopes, account, authority, *args, **kwargs):
         with self.assertRaises(ExpectedBehavior):
             self.app.acquire_token_silent(["different scope"], self.account)
 
+
+class TestApplicationForClientCapabilities(unittest.TestCase):
+
+    @classmethod
+    def setUp(self):
+        self.authority_url_in_app = "https://login.microsoftonline.com/common"
+        self.scopes = ["s1", "s2"]
+        self.client_id = "my_app"
+        self.claims = "{\"id_token\": {\"auth_time\": {\"essential\": true}}}"
+
+    def test_merged_claims_returned_correctly(self):
+        app = ClientApplication(
+            self.client_id,
+            authority=self.authority_url_in_app, client_capabilities=["llt", "ssm"])
+        merged_claims = "{\"id_token\": {\"auth_time\": {\"essential\": true}}, \"access_token\": {\"xms_cc\": {\"values\": [\"llt\", \"ssm\"]}}}"
```

Comment:
My Python is rusty so I could be reading it wrong, but the format of the merged_claims value here looks correct, based on the values of client_capabilities and self.claims that I think get used in the test.

Check out the Claims/ClientCapabilities/ClientCapabilitiesAndClaimsJson values in [this constants file](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/tests/Microsoft.Identity.Test.Common/TestConstants.cs) of the .NET tests, or the very bottom of [this test](https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/dev/src/test/java/com/microsoft/aad/msal4j/AuthorizationRequestUrlParametersTest.java) in the Java tests for other examples of the claims/client capabilities input and the expected output of merging the two together.

Reply:
Yes I referred to these test cases as well, however both of these don not add  `xms_cc` part as a value in id_token field, I might be reading it wrong but I see in the above constants you mentioned we end up adding it only to `access_token` . However, as @rayluo  mentioned, in the [SSM](https://microsoft.sharepoint-df.com/:w:/t/SmartSessionManagementforOfficeClients/EU-SBJAwblNIlqlRwSFFDGUBbZng0DvLJAKNNB8Wy8hBKA?e=bKoiwM) doc mentioned it does say if you need both id_token and the access_token it needs to be added in both of these values. But from the client_capabilities [spec](https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2FClientCapabilities%2Foverview.md&version=GBdev&_a=preview) we only need to add it as an access_token claim so this probably should be clarified. But if you do follow client capabilities spec, it only relates xm_cc as access_token claim

Reply:
Documenting current decision: based on an offline mail thread with stakeholders, this topic is not yet fully settled. In such case, we will go with providing this Client Capability behavior for access_token only, because this gives us more flexibility when/if we will also introduce this to id_token at a later time.